backend/controllers/__test__/chatController.test.js

```diff
@@ -257,15 +258,11 @@ describe("chatController", () => {
   it("should throw an error for an invalid timestamp format", async () => {
     let error;
     try {
-      const user = await User.create({});
-      const chat = await Chat.create({
-        title: "Test Chat",
-        messages: [],
-        polls: [],
-        users: [user],
-      });
-      const invalid_timestamp = "invalid-date";
-      await load_chat(chat._id, invalid_timestamp);
+        const user = await UserService.create_guest_user();
+        const pin = await generate_unique_pin();
+        const chat = await Chat.create({ title: "Test Chat", pin: pin, messages: [], polls: [], users: [user]});
```

Comment:
How does this line work? 
```javascript
.create({ title: "Test Chat", pin: pin, message: [], polls: [], users: [user]});
```
`users` should expect an ObjectId, so it should be `users: [user._id]` here, no? Not sure how this didn't error.

Reply:
yeah you're right